akka-bbb-fsesl/project/Dependencies.scala

```diff
@@ -7,18 +7,18 @@ object Dependencies {
 
   object Versions {
     // Scala
-    val scala = "2.12.8"
+    val scala = "2.12.15"
     val junitInterface = "0.11"
     val scalactic = "3.0.3"
 
     // Libraries
-    val akkaVersion = "2.5.19"
-    val akkaHttpVersion = "10.1.4"
-    val logback = "1.2.3"
+    val akkaVersion = "2.6.17"
+    val akkaHttpVersion = "10.2.7"
+    val logback = "1.3.0-alpha10"

```

Comment:
`val logback = "1.3.0-alpha10"` <- is this production ready? Do we need anything specific that would justify an alpha? Is this related to the log4j security in any way?

Reply:
Yeah makes sense. 
I have email the logback team enquiring about the same question. For now, I have changed the version to [1.2.10] which is their latest stable release.